iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/topology/RemoveFeederBayTest.java

```diff
@@ -99,7 +99,7 @@ void testRemoveWithShunt() {
 
         new RemoveFeederBay("load2").apply(network);
 
-        Set<String> removedIdentifiables = Set.of("SW1", "load2", "load2_DISCONNECTOR", "load2_DISCONNECTOR_6_1", "load2_DISCONNECTOR_6_2", "load2_BREAKER");
```

Comment:
Is this working without the naming strategy?

Reply:
Yes, the naming changed because the logic changed (every disconnector is created the same way while before, the disconnector on the aimed busbar section was created separately, hence the different name). It has nothing to do with the future naming strategy :)